api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -230,6 +231,13 @@ message TransformationTemplate {
   // "my-extractor" extractor.
   map<string, Extraction> extractors = 2;
 
+  // Data sources. use the `data_source`
+  // function to access data_sources in the template (e.g. '{{
+  // data_source("my_file") }}').
+  map<string, envoy.config.core.v3.DataSource> data_sources = 17;
+  // max size of data source. 4096 bytes by default.
+  uint32 max_size = 18;
```

Comment:
Can we name this `data_source_max_size` so it's more clear what it specifically applies to